_drafts/oryx/technology.md

```diff
@@ -0,0 +1,70 @@
+# Oryx technologies
+
+Oryx is a frontend framework that is designed to help developers build web applications using modern technologies and leveraging (Spryker API)[https://docs.spryker.com/docs/scos/dev/glue-api-guides/202307.0/decoupled-glue-api.html].
+
+Oryx leverage standard technology, frameworks and tools provided by the web platform and the community wherever possible. It emphasizes utilizing the inherent capabilities of the web and minimizing the reliance on build tools. During development, Oryx primarily employs JavaScript but aims to reduce the need for executing JavaScript in the browser. It prioritizes HTML and CSS for delivering a fast-loading experience across various screen sizes.
```

Comment:
Is `wherever possible` related to only `the community` or to all the items in the sentence?